backend/app/controllers/spree/admin/orders_controller.rb

```diff
@@ -11,6 +11,7 @@ def index
         params[:q][:completed_at_not_null] ||= '1' if Spree::Config[:show_only_complete_orders_by_default]
         @show_only_completed = params[:q][:completed_at_not_null] == '1'
         params[:q][:s] ||= @show_only_completed ? 'completed_at desc' : 'created_at desc'
+        params[:q][:completed_at_not_null] = '' unless @show_only_completed
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.